cypress/support/BonnieFHIR/BonnieUploadMeasure.js

```diff
@@ -52,6 +52,7 @@ export const UploadMeasureToBonnie = (fileToUpload, calculation, vsacLoggedIn) =
   helper.enabled(importMeasureDialog.importLoadBtn)
   helper.click(importMeasureDialog.importLoadBtn)
 
+  cy.wait(2000)
```

Comment:
Looks like some latency was introduced with new jQuery

Reply:
I think this is fine for now, but we might want to see if we can wait for an specific api call to load similar to what we are doing for vsac api